lib/dom/dom.nit

```diff
@@ -67,3 +67,27 @@ redef class XMLStartTag
 		abort
 	end
 end
+
+redef class XMLAttrTag
+
+	# Attributes as a map (ignoring malformed attributes)
```

Comment:
Since attributes should be unique in well-formed XML, I would prefer to not document this behavior.